src/components/ble/BleController.cpp

```diff
@@ -2,12 +2,12 @@
 
 using namespace Pinetime::Controllers;
 
-void Ble::Connect() {
-  isConnected = true;
+void Ble::SetConnectState(Ble::ConnectStates newState) {
```

Comment:
I'm not sure we should mix "connection state" (connected and disconnected are implemented, and other future states could be "error", "failed", "refused",...) with an "operating mode" (normal and airplane mode).
I think they are 2 distinct concept that should be kept separate : 
 - ConnectionMode : What is the status of the BLE connectivity : are we connected or not ?
 - OperatingMode (or ConnectionAllowed, or something similar?) : Do we accept new BLE connections ?

What do you think?